featuretools/variable_types/api.py

```diff
@@ -2,6 +2,7 @@
 from .utils import (
     find_variable_types,
     graph_variable_types,
+    list_logical_types,
```

Comment:
Are we eventually going to get rid of the `variable_types` directory? If so, we probably need somewhere else to put these. 

This also makes me wonder if we're going to need equivalents of `graph_variable_types` and `find_variable_types` (@gsheni )? We might be able to make use of the Woodwork type system's parent-child relationships, but idk

Reply:
We can completely remove `find_variable_types` and `graph_variable_types`. With WW we want to move away from the parent-child relationship. 

Yes, we should get rid of the variable types directory.

Reply:
Do we want users to import `from featuretools.logical_types import list_logical_types`?

Reply:
If we don't need to have a logical_types folder, let's not have it.`from featuretools import list_logical_types` seems good to me.

Reply:
Do we want to keep the reference in `featuretools.variable_types.utils` for now and make it possible to import from the top-level `from featuretools import list_logical_types`?

Reply:
My vote would be no. There is already going to be lots of breaking changes, and we can just be sure to note them in the release notes, and add it to the documentation page for V1 upgrade.

Reply:
Okay, so we want to import `from featuretools import list_logical_types` and wait to see where to move this function when we delete variable types in a future PR right?

Reply:
I can also just add another import statement to the reference in `featuretools.__init__`

Reply:
@jeff-hernandez its your call. If you want to do it in a followup MR, just be sure to create an issue. If you want to do it in this MR, that is also fine.

Reply:
Okay, i'll add another import statement at the top-level